ddl/column.go

```diff
@@ -31,15 +31,20 @@ import (
 	"github.com/pingcap/tidb/util/types"
 )
 
-func (d *ddl) adjustColumnOffset(columns []*model.ColumnInfo, indices []*model.IndexInfo, offset int, added bool) {
+func (d *ddl) adjustColumnOffset(columns []*model.ColumnInfo, indices []*model.IndexInfo, offset int, lastColOffset int, added bool) {
```

Comment:
It should be the least offset for all new added columns and it will always be right when it is 0. It seems we needn't consider the `offset` when we add a column.